APIs/openEO/large_scale_processing.qmd

```diff
@@ -59,40 +57,40 @@ df = gpd.read_file("https://artifactory.vgt.vito.be/auxdata-public/grids/LAEA-20
 
 ## Prepare job attributes
 
-In addition to the tiling grid, we recommend determining other necessary properties for your processing jobs in advance. This enables a thorough review of these properties before initiating the processing. Examples includes basic elements such as job titles or tile-specific processing parameters, as well as attributes to determine the processing order.
+In addition to defining the tiling grid, it is advisable to pre-determine other essential properties required for processing jobs. This approach allows for a comprehensive review of these properties before initiating processing. Examples include fundamental elements like job titles or tile-specific processing parameters, as well as attributes necessary to establish the processing sequence.
 
-In this step, you may also want to make sure to determine the correct tile extent in the coordinate system of your tile grid. Providing exact coordinates in the right projection is necessary to ensure pixel-perfect alignment of your tiles.
+During this phase, users may also need to establish the precise tile extents within the coordinate system of their chosen tile grid. Providing accurate coordinates in the correct projection ensures precise alignment of tiles at the pixel level.
 
 ## Tuning your processing job
 
-Before kicking off large processing, you want to be very sure that the correct output is generated and that you have sufficient credits and resources available to finish your job in time. This can be done by simply running various jobs and using the statistics reported in the metadata to determine average parameters. (The map production section below shows a way to collect these parameters in a CSV.)
+Before starting large-scale processing, users should verify that the correct output is generated and that they have sufficient credits and resources to complete their jobs. This can be achieved by running test jobs and analyzing the statistics reported in the metadata to determine average parameters. (Refer to the map production section below for guidance on collecting these parameters in a CSV format.)
 
-For instance, for the case of processing the EU27 croptype map, consisting of ~11000 20km tiles, we made the following calculations up front:
+For example, when processing the EU27 croptype map, consisting of ~11000 20km tiles, we conducted the following upfront calculations:
 
-- The average runtime was 30 minutes, which means that it would take ~15 days of continuous processing with 15 parallel jobs.
-- The average cost was below 100 credits, so we would be able to process with a budget of 1100000 credits.
+- The average runtime per job was 30 minutes, indicating it would require approximately 15 days of continuous processing with 15 parallel jobs.
```

Comment:
Is this math correct for the standard 2 concurrent jobs?

I think it would result in +200 days